.golangci.yaml

```diff
@@ -1,29 +1,140 @@
+# Based on recommendations in https://olegk.dev/go-linters-configuration-the-right-version
+
 run:
+  # linter execution
   timeout: 30m
+  issues-exit-code: 1
+  concurrency: 4
+  # check tests as well
+  tests: true
+  # fail if go.mod file is outdated.
+  modules-download-mode: readonly
+  # use the Go version from the go.mod file.
+  go: ""
 
 linters:
-  disable-all: false
+  # set to true to run only fast linters (e.g., for pre-commit)
+  fast: false
+
+  disable-all: true
   enable:
+    - asasalint
+    - asciicheck
+    - bidichk
     - bodyclose
-    - errcheck
+    - contextcheck
+    - durationcheck
+    - errcheck 
+    - errname
+    # - errorlint
```

Comment:
I think you should  open a PR for all the linters you want to add in the future but wasn't able to do it now

Reply:
A PR would involve making all of the code changes to make each linter pass. Did you mean an issue? I opened #61 to track this wholesale, with needed linters already in this PR's config file only commented out.
Do you think anything else is needed?

Reply:
That's ok